src/node.Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM docker.io/cimg/node:current-browsers
 WORKDIR /app
 
+# Install gosu
+USER root
+RUN apt-get update && \
```

Comment:
Comment this block a bit more.
Where is gosu used?

Reply:
Added comment. Gosu used in entrypoint to run command as circleci user